RS-Project2/ShoppingCart/PromotionOrder.java

```diff
@@ -0,0 +1,142 @@
+package ShoppingCart;
+
+import java.util.*;
+import java.text.SimpleDateFormat;
+import java.util.*;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+enum PromotionEnum {
+    PromoCode1("FAAVC", "22/05/2017", "22/12/2018"), PromoCode2("FAAVD", "22/06/2018", "22/12/2018"), PromoCode3("FAAVE", "01/06/2018",

```

Comment:
Line Length Isssue

Reply:
The variable name should not be as 1,2,3 - The variable name should be relavent.